src/data/events.ts

```diff
@@ -76,12 +76,27 @@ export const EVENTS: Event[] = [
         date: { year: 2024, month: 'MAY', day: 10, endTime: '23:00' },
         time: '5:30pm - 11:00pm',
         location: 'UniBar Adelaide and various bars in Adelaide',
-        details: 'Our annual pub crawl! More details to come soon.',
-        url: {
-            href: new URL('https://csclub-adl.square.site/product/cs-club-pub-crawl-2024-shirt/54'),
-            text: 'Preorder shirts now!',
-        },
-        image: 'upcoming-event.jpg',
+        details:
+            'Join us for a night of hopping from one watering hole to the next, spreading good vibes and great drinks all around.',
+        image: 'pub-crawl.png',
+    },
+    {
+        title: 'Meet and Greet',
+        date: { year: 2024, month: 'JUL', day: 26, endTime: '21:00' },
+        time: '5:00pm - 9:00pm',
+        location: 'Engineering and Maths 110 & 105',
```

Comment:
```suggestion
        location: 'Engineering and Maths EM110 & EM105',
```

Room names are EM110 and EM105